test_runner/regress/test_sharding_service.py

```diff
@@ -938,3 +938,67 @@ def storage_controller_consistent():
         env.storage_controller.consistency_check()
 
     wait_until(10, 1, storage_controller_consistent)
+
+
+def test_sharding_service_re_attach(neon_env_builder: NeonEnvBuilder):
+    """
+    Exercise the behavior of the /re-attach endpoint on pageserver startup when
+    pageservers have a mixture of attached and secondary locations
+    """
+
+    neon_env_builder.num_pageservers = 2
+    env = neon_env_builder.init_start()
+
+    # Don't want the default tenant: we'll use explicitly created tenants with a known placement policy
+    env.storage_controller.pageserver_api().tenant_delete(env.initial_tenant)
```

Comment:
You could do the following to avoid the default tenant being created:
```
env = neon_env_builder.init_configs()
env.start()
```

Reply:
Very true.  e389cd74f4f44bf581439228d06f55888dc73ceb